categories.go

```diff
@@ -127,6 +128,24 @@ func newCategoriesFilterFromQuery(query url.Values) (*packages.Filter, error) {
 		}
 	}
 
+	if v := query.Get("capabilities"); v != "" {
+		filter.Capabilities = strings.Split(v, ",")
+	}
+
+	if v := query.Get("spec.min"); v != "" {
+		filter.SpecMin, err = semver.NewVersion(v)
+		if err != nil {
+			return nil, fmt.Errorf("invalid 'spec.min' version '%s': %w", v, err)
+		}
+	}
+
+	if v := query.Get("spec.max"); v != "" {
+		filter.SpecMax, err = semver.NewVersion(v)
+		if err != nil {
+			return nil, fmt.Errorf("invalid 'spec.max' version '%s': %w", v, err)
+		}
+	}
+
```

Comment:
Main changes included in this PR, adding these new filters to categories endpoint.